src/oskbd/windows/interception.rs

```diff
@@ -149,7 +149,7 @@ thread_local! {
     static INTRCPTN: Interception = Interception::new().expect("interception driver should init: have you completed the interception driver installation?");
 }
 
-#[cfg(not(feature = "simulated_output"))]
+#[cfg(any(not(feature = "simulated_output"), not(feature = "passthru_ahk")))]
```

Comment:
Looks like I missed some, but this happens to not conflict with the tested set of feature flags anyway... but this and line 174 should probably be `all`